modules/user/user.service.js

```diff
@@ -132,7 +145,7 @@ class UserService {
       ],
     });
     const savedUser = await user.save();
-    return savedUser._doc;
+    return { ...savedUser._doc, id: savedUser._id };
```

Comment:
Can you just return `savedUser`? This code looks weird

Reply:
fixed